processor-kotlin/build.gradle.kts

```diff
@@ -11,14 +11,21 @@ tasks.withType<Test> {
 }
 
 dependencies {
+    val jacksonVersion = "2.12.2"
```

Comment:
Do we want to define dependencies in the build.gradle.kts, or do we want to define them via gradle.properties (accessing via `val jacksonVersion: String by project`)? Not necessarily something that needs to be changed, but we should keep this in mind for the future.

Reply:
Honestly... idc as both ways are just fine. And those are not necessarily advantages over each other

Reply:
In that case, let's stick with what we've got right now :)